jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/Content.java

```diff
@@ -327,18 +331,34 @@ default long getLength()
         void demand(Runnable demandCallback);
 
         /**
-         * <p>Fails this content source, possibly failing and discarding accumulated
-         * content chunks that were not yet read.</p>
+         * <p>Fails this content source with a terminal {@link Chunk#getError() error},
```

Comment:
A _terminal error_ sounds unclear, I would prefer to read that an error with the last flag set to true will be returned by the next read.